cms/static/js/views/settings/main.js

```diff
@@ -151,6 +162,22 @@ var DetailsView = ValidatingView.extend({
         case 'course-effort':
             this.setField(event);
             break;
+        case 'entrance-exam-enabled':
+            if($(event.currentTarget).is(":checked")){
+                this.$el.find('#div-grade-requirements').show();
+            }else{
+                this.$el.find('#div-grade-requirements').hide();
+            }
+            this.setField(event);
+            break;
+        case 'entrance-exam-minimum-score-pct':
+            // If the val is an empty string then update model with default value.
+            if($(event.currentTarget).val() == ''){
+                this.model.set('entrance_exam_minimum_score_pct', this.model.defaults.entrance_exam_minimum_score_pct);
```

Comment:
It would be good to add a unit test to verify that this defaulting works as expected.